README.md

```diff
@@ -68,38 +70,38 @@ the steps accordingly.
 
 ### Database setup
 
-1.  *ff-node-monitor* needs PostgreSQL as a database backend:
+6.  *ff-node-monitor* needs PostgreSQL as a database backend:
 
     ```
     sudo apt install postgresql
     ```
 
-2.  We will use the `ff-node-monitor` system user to access PostgreSQL, and we
+7.  We will use the `ff-node-monitor` system user to access PostgreSQL, and we
     need to create a database for the service:
 
     ```
     sudo -u postgres psql -c 'CREATE ROLE "ff-node-monitor" WITH LOGIN;'
     sudo -u postgres psql -c 'CREATE DATABASE "ff-node-monitor" WITH OWNER = "ff-node-monitor" LC_COLLATE = '\''de_DE.utf8'\'' TEMPLATE template0;'
     ```
 
-    You may have to install the `de_DE.UTF-8` locale before this works.  On
+    You may have to install the `de_DE.UTF-8` locale before this works. On
```

Comment:
I am consistently using two spaces after a full stop. Please keep that style.

Reply:
why did you choose this style with two spaces? It is not known to me, that this is correct. (Seems weird)

Reply:
>  So could you remove the last commit?

done

Reply:
> But why did you choose this style with two spaces? It is not known to me, that this is correct. (Seems weird)

It is what emacs uses when you let it auto-wrap text. It matches the good typesetting practice to make the space after a full stop bigger than the space between words, increasing readability (any good typesetting system will do that). Usually the space shouldn't be twice as big, but there's only so much you can do with a fixed-width font.